packages/components/organisms/f-checkout/src/mixins/logger.mixin.js

```diff
@@ -0,0 +1,18 @@
+function buildErrorLogFields (error) {
+    return {
+        ...(error && error instanceof Error && {
+            exception: error.name,
+            exceptionMessage: error.message,
+            exceptionStackTrace: error.stack
+        })
+    };
+}
+
+export default {
+    methods: {
+        logInvoker (message, data, callback, error) {
```

Comment:
I think as we discussed (I might be wrong), I'd make the parameter an object instead: `logInvoker ({ message, data, callback, error }) { ... }`